pages/index.vue

```diff
@@ -821,13 +838,24 @@
             };
             this.$refs.historyComponent.addEntry(entry);
             return;
+          } else {
+            this.response.status = error.message;
+            this.response.body = "See JavaScript console (F12) for details.";
+            this.$toast.error('Something went wrong!', {
+              icon: 'error'
+            });
+            if(!this.$store.state.postwoman.settings.PROXY_ENABLED) {
+              this.$toast.info('Turn on the proxy', {
```

Comment:
This is looking good to me, my only recommendation is that this be renamed to 'Try enabling the proxy' or words to that effect, but it's not that significant.

Reply:
How about "Switch to Proxy mode"?

Reply:
Yes - but perhaps with the question mark. The idea being to imply to the user that it's something that might help with the error.